src/exampleTypes/Geo.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { observer } from 'mobx-react'
+import { Flex } from '../layout/Flex'
+import { exampleTypes } from 'contexture-client'
+import injectTreeNode from '../utils/injectTreeNode'
+
+const customStyles = {
+  valueContainer: styles => ({
+    ...styles,
+    ...{
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
+    },
+  }),
+}
+
+const elementStyle = {
+  flex: 1,
+  marginBottom: '5px',
+}
+
+const operatorOptions = ['within', 'not within']
+
+let selectInput = props => <select {...props}></select>
```

Comment:
I ran `npm run lint` in the repo and this caused an error, it said that empty components should auto close 👍

Reply:
ok

Reply:
Added the async/await as per Daniel. Had to add additional `await` since `fetch` .json() also needs to be awaited.